source/_components/nissan_leaf.markdown

```diff
@@ -0,0 +1,132 @@
+---
+layout: page
+title: "Nissan Leaf"
+description: "Instructions for how to integrate Nissan Leaf(s) into Home Assistant."
+date: 2019-01-05 07:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: nissan.png
+ha_category: Car
+ha_release: 0.88
+ha_iot_class: "Cloud Polling"
+redirect_from:
+ - /components/binary_sensor.nissan_eaf/
```

Comment:
```suggestion
 - /components/binary_sensor.nissan_leaf/
```

possibly that all these redirects are not necessary because the pages did not exist from the beginning.

Reply:
Thanks.  Removed all the redirects in commit https://github.com/home-assistant/home-assistant.io/pull/8073/commits/56c5dd93bbcdc4e6bceebc9b1828c6a1671ea641